Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.6.5
+RUN apt-get update && apt-get install -y default-mysql-client
+
+WORKDIR /otwa
+COPY Gemfile .
+COPY Gemfile.lock .
+RUN gem install bundler -v 1.17.3 && bundle install
+
+COPY . .
```

Comment:
I don't believe we need to copy the app into the image because we're mounting it as a volume at runtime.

The rake tasks in `docker-init.sh` don't run that way, but I think that's just because the volume isn't mounted yet. That can probably be avoided by starting the web service (thereby mounting the volume) before running the rake tasks.